### ISSUE TITLE
logKextClient states version 2.2

### ISSUE THREAD

**Issue by laderbogen** [SlEePlEs5/logKext#6]
But I did the install for version 2.4 of logKext, is this correct?
Can the client be of an other version than logKext?

regards

**Comment by SlEePlEs5** [comment]
Yes, this is version 2.4. This mistake was already present in logKext 2.3, where a version string was erroneously left as 2.2. This will be fixed soon, along with some other issues, but it has a low priority, as it does not affect the functionality of the program.
